javascript-forms/form-events.js

```diff
@@ -0,0 +1,53 @@
+function handleFocus(event) {
+  console.log('focusEvent: fired');
+  console.log('targetName:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blurEvent: fired');
+  console.log('targetName:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log(
+    'targetName:', event.target.name,
+    'targetValue:', event.target.value);
+}
+
+function formReset() {
+  document.getElementById('contact-form').reset();
+}
+
+function onSubmit(event) {
+  event.preventDefault();
+  var name = contactForm.elements.name.value;
+  var email = contactForm.elements.email.value;
+  var textArea = contactForm.elements.message.value;
+  var formData = {};
+  formData.name = name;
+  formData.email = email;
+  formData.textArea = textArea;
+
+  formReset();
+  console.log(formData);
+}
```

Comment:
The form submit stuff should go in `form-values.js`, not `form-events.js`:

---

![image](https://user-images.githubusercontent.com/16143771/178861906-7911dff1-e813-40d2-8265-04178a447e9a.png)

Reply:
done!